### PR TITLE
Add a test RP to test navigator.id.get

### LINKED ISSUE #3507: No way to test navigator.id.get from the dev environment.

### PR DESCRIPTION
**Pull request by shane-tomlinson**
fixes #3507

### CODE REVIEW

File: example/rp/get.html [hunk 1]
```diff
@@ -0,0 +1,163 @@
+<!DOCTYPE html>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+
+<html>
+<head>
+<meta charset="utf-8">
+<meta name="viewport" content="initial-scale=1.0; maximum-scale=1.0; width=device-width;">
```
Comment (callahad) [inline example/rp/get.html]: Why the `maximum-scale=1.0`?

File: example/rp/get.html [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+function checkAssertion(assertion) {
+  $.ajax({
+    url: "/process_assertion",
+    type: "post",
+    dataType: "json",
+    data: {
+      assertion: assertion,
+      audience: window.location.protocol + "//" + window.location.host,
+      forceIssuer: $('#forceIssuer').attr('checked') ? "issuer.domain" : undefined,
+      allowUnverified: $('#allowUnverified').attr('checked') ? true : false
```
Comment (callahad) [inline example/rp/get.html]: I thought these two weren't in `.get`? And there's no UI for them on the page.

**Comment by callahad** [comment]
I'm r+, minus those two nits. Feel free to resolve (or not) and then merge.

File: example/rp/get.html (hunk 1 above)
Comment (shane-tomlinson) [inline example/rp/get.html]: This is a copy of index.html - a few features.

File: example/rp/get.html (hunk 2 above)
Comment (shane-tomlinson) [inline example/rp/get.html]: Good catch, I will remove those.

**Pull request merged**